_data/scout-cli/docker_scout_compare.yaml

```diff
@@ -0,0 +1,212 @@
+command: docker scout compare
+short: '[early preview] Compare two images and display differences'
+long: |-
+    The `docker scout compare` command analyzes two images and display a comparison of both.
+
+    > This command is in **early preview** and its behaviour might change in the future
+
+    The main usage is to compare two versions of the same image.
+    For instance when a new image is built and compared to the version running in production.
+
+    The following artifact types are supported:
+
+    - Images
+    - OCI layout directories
+    - Tarball archives, as created by `docker save`
+
+    The tool analyzes the provided software artifact, and generates a vulnerability report.
+
+    By default, the tool expects an image reference, such as:
+
+    - `redis`
+    - `curlimages/curl:7.87.0`
+    - `mcr.microsoft.com/dotnet/runtime:7.0`
+
+    If the artifact you want to analyze is an OCI directory or a tarball archive, you must use the `--type` or `--to-type` flag.
+usage: docker scout compare --to IMAGE|DIRECTORY|ARCHIVE IMAGE|DIRECTORY|ARCHIVE
+pname: docker scout
+plink: docker_scout.yaml
+options:
+    - option: debug
+      value_type: bool
+      default_value: "false"
+      description: Debug messages
+      deprecated: false
+      hidden: true
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: format
+      value_type: string
+      default_value: text
+      description: |-
+        Output format of the generated vulnerability report:
+        - text: default output, plain text with or without colors depending on the terminal
+        - markdown: Markdown output
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: ignore-base
+      value_type: bool
+      default_value: "false"
+      description: Filter out CVEs introduced from base image
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: only-fixed
+      value_type: bool
+      default_value: "false"
+      description: Filter to fixable CVEs
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: only-package-type
+      value_type: stringSlice
+      default_value: '[]'
+      description: |
+        Comma separated list of package types (like apk, deb, rpm, npm, pypi, golang, etc)
```

Comment:
```suggestion
        Comma separated list of package types (e.g. apk, deb, rpm, npm, pypi, golang, etc)
```

Reply:
Same, I'm tempted to keep it for now and then have a pass over all the scout docs, so it's consistent across all the commands.